codec_registry.src.html

```diff
@@ -125,22 +122,23 @@
   </tr>
   <tr>
     <td>ulaw</td>
-    <td>PCM u-law</td>
-    <td>N/A</td>
+    <td>u-law PCM</td>
```

Comment:
A minor thing, should we refer to this as μ-law, to be more precise (here and in the registration entry)?

Reply:
I'd like to keep the actual string used in code as 'ulaw' (using unicode chars in code seems burdensome), so I'm worried that using the μ anywhere may lead to confusion and copy/paste

I agree μ-law is more precise. FWIW, I do find some precedent for ulaw
https://developer.apple.com/library/archive/documentation/QuickTime/QTFF/QTFFChap3/qtff3.html
http://mp4ra.org/#/codecs